src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/ExceptionRecordFinalizerTest.java

```diff
@@ -9,16 +10,18 @@
 import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
 import static org.assertj.core.api.Assertions.assertThat;
-import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.CASE_REFERENCE;
-import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.DISPLAY_WARNINGS;
-import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.OCR_DATA_VALIDATION_WARNINGS;
 import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.YesNoFieldValues.NO;
 
 class ExceptionRecordFinalizerTest {
 
     private static final long CASE_ID = 100L;
     private static final String FIELD_1 = "field1";
 
+    public static final String CASE_REFERENCE = "caseReference";
+    public static final String DISPLAY_WARNINGS = "displayWarnings";
+    public static final String OCR_DATA_VALIDATION_WARNINGS = "ocrDataValidationWarnings";
+    public static final String ATTACH_TO_CASE_REFERENCE = "attachToCaseReference";
```

Comment:
We can't reuse constants from the main code when verifying map entries. Otherwise, we'd be checking if "the code sets the entries that it sets".